packages/akashic-cli-export/src/zip/licenseUtil.ts

```diff
@@ -40,14 +40,23 @@ async function makeLicenseInfo(source: string, pkgJsonPaths: string[]): Promise<
     for (const p of pkgJsonPaths) {
         const pkgJsonPath = path.resolve(source, p);
         const pkgJson = await cmn.FileSystem.readJSON<any>(pkgJsonPath);
-        // LICENSE ファイルは root 直下のファイルを部分一致で取得
-        // TODO: LICENSE-LGPL のようなファイルは機械的に扱えないので警告を出すようにする。 LICENSE(license) に任意の拡張子がついたものを対象とする。
+
         const files = fs.readdirSync(path.dirname(pkgJsonPath));
-        const licenseFile = files.find((file) => /LICENSE/.test(file));
-        if (!licenseFile) return;
+        const licenseFile = files.find((file) => {
+            // 仕様上はあらゆる拡張子が許されているが、テキストファイルに取り込むのでここでは .txt と .md のみ考慮する
+            if (/^LICEN[SC]E$/i.test(file) || /^LICEN[SC]E.(txt|md)$/i.test(file)) {
+                return true;    
+            }
+            // LICENSE-LGPL のようなファイルは機械的に扱えないので警告を出力
+            if (/LICENSE\-\w+$/i.test(file)) {
+                console.warn(`[WARNING]: The license of ${pkgJson.name} will not be included in thirdparty_license.txt since akashic export doesn't know its license file name "${file}".You may need to follow the license by yourself.`);
```

Comment:
これ LICENSE-LGPL と LICENSE が両方あるケースでは「この警告で "will not included" と出力した後に、LICENSE が見つかって普通に include される」みたいなことが起きませんか？ (readdirSync() の返す値の順序に依存しますが、[仕様](https://nodejs.org/api/fs.html#fsreaddirsyncpath-options) から言及されている POSIX の readdir(3) はファイルシステム依存だと [述べています](https://man7.org/linux/man-pages/man3/readdir.3.html))

単に無視するか、でなければ `[WARNING]: Detected a license-like file "<file>" in ${pkgJson.name} but ignored (not included in thirdparty_license.txt) because akashic export doesn't know how it should be handled. You may need to follow the license by yourself.` みたいなメッセージの方が良さそうです。

Reply:
ありがとうございます。メッセージの方を修正しました。